client/src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,130 @@
+import { useEffect } from 'react';
+import { NavLink } from 'react-router-dom';
+
+import CssBaseline from '@material-ui/core/CssBaseline';
+
+// Context APIs
+import { useAuth } from '../../context/useAuthContext';
+import { useSocket } from '../../context/useSocketContext';
+
+// Material
+import {
+  Toolbar,
+  AppBar,
+  Box,
+  CircularProgress,
+  Typography,
+  Button,
+  Badge,
+  Link as MaterialLink,
+} from '@material-ui/core';
+import { Notifications } from '@material-ui/icons';
+
+// Supporting
+import AuthMenu from '../AuthMenu/AuthMenu';
+import Logo from '../../Images/logo.png';
+
+// Styles
+import useStyles from './useStyles';
+
+function LoggedInBar({ classes, loggedInUser }: any): JSX.Element {
+  return (
+    <Box>
+      <NavLink to="/dashboard" className={classes.link}>
```

Comment:
You can combine react-router's Link with the MUI Button :)

Ref: https://stackoverflow.com/questions/51642532/how-to-make-a-material-ui-react-button-act-as-a-react-router-dom-link